trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/untrimmed.py

```diff
@@ -1,18 +1,14 @@
-## @ingroup Methods-Aerodynamics-Fidelity_Zero-Drag
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Drag
```

Comment:
Do a compare of the texts. They are different enough to justify separate scripts.